TouchSky/TouchSky/TouchSky/Views/PeakAnnotationView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PeakAnnotationView.swift
+//  TouchSky
+//
+//  Created by Alexa Nicole Hugenberg on 10/17/22.
+//
+
+import SwiftUI
+
+struct PeakAnnotationView: View {
+    
+    @State private var showTitle = true
+    
+    let title: String
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            Text(title)
+                .font(.callout)
+                .padding(5)
+                .background(Color(.white))
+                .cornerRadius(10)
+                .opacity(showTitle ? 0 : 1)
+            
+            Image(systemName: "mappin.circle.fill")
+                .font(.title)
+                .foregroundColor(.red)
+            
+            Image(systemName: "arrowtriangle.down.fill")
+                .font(.caption)
+                .foregroundColor(.red)
+                .offset(x: 0, y: -5)
+        }
+        .onTapGesture {
+            withAnimation(.easeInOut) {
+                showTitle.toggle()
```

Comment:
I'm a little bit confused about the behavior of tapping pins in the map view. It seems sometimes it would show the title and sometimes it would push the view from the `NavigationLink` you setup in `MapView`.

You mentioned that:

> A single tap on a pin brings up the name of the 14er and a double tap brings up more information about the 14er

But I'm not sure if that's actually what the behavior will be here. I think you may need to define separate tap gestures to accomplish this properly. You can set a double tap gesture with a `count` argument. Maybe just doing that will work, I'm not sure.

https://www.hackingwithswift.com/quick-start/swiftui/how-to-read-tap-and-double-tap-gestures